### PR TITLE
javascript-timers - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
https://user-images.githubusercontent.com/106217355/181317970-698a23cc-443f-423f-8c91-be464db547d5.mp4

### CODE REVIEW

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/181318533-b5dad13b-b1c7-4678-bb42-6e53f40c028e.mp4

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var textEl = document.querySelector('.countdown-display');
+
+function changeCount() {
+  if (textEl.textContent === '4') {
+    textEl.textContent = '3';
+  } else if (textEl.textContent === '3') {
+    textEl.textContent = '2';
+  } else if (textEl.textContent === '2') {
+    textEl.textContent = '1';
+  } else if (textEl.textContent === '1') {
```
Comment (codingcodymiller) [inline javascript-timers/interval.js:10]: What if the timer started from 1000? You should try to find a way to do this that doesn't require a conditional for each number.

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    textEl.textContent = '~Earth Beeeelooowww Us~';
+  }
+}
+
+setInterval(changeCount, 1000);
+var interval = setInterval(changeCount, 1000);
```
Comment (codingcodymiller) [inline javascript-timers/interval.js:16]: You are calling `setInterval` twice 🤔

File: javascript-timers/interval.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+clearInterval(interval);
```
Comment (codingcodymiller) [inline javascript-timers/interval.js:17]: `clearInterval` should only be called when you want to stop your interval entirely, not immediately after the interval begins.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**